content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -24,56 +24,34 @@ You might choose to deploy Istio ingress gateways in various network topologies
 Istio ingress gateway to the Internet). As such, these topologies require different ingress gateway configurations for
 transporting correct client attributes like IP addresses and certificates to the workloads running in the cluster.
 
-Configuration of XFF and XFCC headers is managed via `MeshConfig` during Istio
-installation or by adding a pod annotation. Note that the `Meshconfig` configuration is a global setting for all gateway workloads, while pod annotations override the global setting on a per-workload basis.
+Configuration of XFF and XFCC headers can be set globally for all gateway workloads via `MeshConfig` or per gateway using a pod annotation. For example, to configure globally during install or upgrade when using an `IstioOperator` custom resource:
 
-To simplify configuring network topology during installation, create a single YAML file to pass to `istioctl`:
-
-{{< text bash >}}
-$ cat <<EOF > topology.yaml
-apiVersion: install.istio.io/v1alpha1
-kind: IstioOperator
+{{< text yaml >}}
 spec:
   meshConfig:
     defaultConfig:
       gatewayTopology:
-EOF
+        numTrustedProxies: <VALUE>
+        forwardClientCertDetails: <ENUM_VALUE>
 {{< /text >}}
 
-{{< idea >}}
-If Istio ingress gateway was already running prior to application of the `MeshConfig`, restart any Istio ingress gateway pods.
-{{< /idea >}}
-
-You can configure both of these settings using the `proxy.istio.io/config` annotation to the Pod spec
+You can also configure both of these settings by adding the `proxy.istio.io/config` annotation to the Pod spec
 of your Istio ingress gateway.
 
 {{< text yaml >}}
 ...
   metadata:
     annotations:
-      "proxy.istio.io/config": '{"gatewayTopology" : { "numTrustedProxies": 2 } }'
+      "proxy.istio.io/config": '{"gatewayTopology" : { "numTrustedProxies": <VALUE>, "forwardClientCertDetails": <ENUM_VALUE> } }'
 {{< /text >}}
 
 ### Configuring X-Forwarded-For Headers
 
 Applications rely on reverse proxies to forward client attributes in a request, such as `X-Forward-For` header. However, due to the variety of network
-topologies Istio can be deployed in, you must set the number of trusted proxies deployed in front
-of the Istio gateway proxy, so that the client address can be extracted correctly.
+topologies that Istio can be deployed in, you must set the `numTrustedProxies` to the number of trusted proxies deployed in front
+of the Istio gateway proxy, so that the client address can be extracted correctly. This helps the Istio Gateway set the correct value for the `X-Envoy-External-Address` header.
```

Comment:
```suggestion
of the Istio gateway proxy, so that the client address can be extracted correctly.
This controls the value populated by the ingress gateway in the `X-Envoy-External-Address` header
which can be reliably used by the upstream services to access client's original IP address.
```
```suggestion
of the Istio gateway proxy, so that the client address can be extracted correctly. This helps the Istio Gateway set the correct value for the `X-Envoy-External-Address` header.
```